cms/admin/forms.py

```diff
@@ -166,7 +166,7 @@ class PageForm(PageAddForm):
     # moderation - message is a fake field
     moderator_message = forms.CharField(max_length=1000, widget=forms.HiddenInput, required=False)
     
-    redirect = forms.CharField(label=_('Redirect'), max_length=255, required=False,
+    redirect = forms.URLField(label=_('Redirect'), max_length=255, required=False,
```

Comment:
Redirecting to a relative URL is valid as far as HTTP goes, and there are already many relative redirects saved in the database. I don't think we should make this change.